master/docs/conf.py

```diff
@@ -21,7 +21,18 @@
 sys.path.append(os.path.abspath('.'))
 
 # -- General configuration -----------------------------------------------------
-
+try:
+    import sphinxcontrib.blockdiag
```

Comment:
@rutsky what we usually do in the code is  
```
import sphinxcontrib.blockdiag
[sphinxcontrib.blockdiag]
```

problem with assert is that it evaluates the Trueness of the thing which could actually do some side effects or even return false